src/CreateHandler/CreateHandlerCommand.php

```diff
@@ -59,9 +59,13 @@ class. For a path, it matches the class namespace against PSR-4 autoloader
      */
     protected function configure() : void
     {
+        $this->handlerArgument = (bool) preg_match('/^action:/', $this->getName())
```

Comment:
We don't need to use regexp, `strpos === 0` is enough.

Reply:
I can't see property `handlerArgument`, should be added instead of dynamic creation.

Reply:
It's needed withing the `execute()` method, so that we pull the correct argument.

Reply:
Yes, but what I ment is that property `handlerArgument` is not defined in the class (it is created here dynamically what is not a good practice) and I've suggested to use `strpos` instead of regular expression, because there is no reason to have the regexp here. See my comment below also how I would change it with property:
```php
protected $handlerArgument = 'handler';
```
(or `private`).